canonical/cs.canonical.com
# [WD-17923] split routes into several files

## Done

Split routes defined in app.py into several files for better readability and maintainability

## QA

- Open the demo
- Check that everything works as expected

## Issue

Fixes https://warthogs.atlassian.net/browse/WD-17923
